client/state/activity-log/actions.js

```diff
@@ -159,6 +160,25 @@ export function rewindRestore( siteId, timestamp, args ) {
 	};
 }
 
+/**
+ * Restore specific files and/or database tables to the given timestamp.
+ *
+ * @param {string|number} siteId the site ID
+ * @param {string|number} timestamp Unix timestamp to restore site to
+ * @param {string} includePaths List of included files, separated by comma
+ * @param {string} excludePaths List of excluded files, separated by comma
+ * @returns {Object} action object
+ */
+export function rewindGranularRestore( siteId, timestamp, includePaths, excludePaths = '' ) {
+	return {
+		type: REWIND_GRANULAR_RESTORE,
+		siteId,
+		timestamp,
+		includePaths,
+		...( excludePaths ? { excludePaths } : {} ),
```

Comment:
This change is addressing https://github.com/Automattic/wp-calypso/pull/81326#pullrequestreview-1608716491:

> The restore logs mention one included path and one excluded path. I'm not sure if this is because we're passing an empty excluded path. We may be expected to trim it and only pass excluded paths if they're non-empty. I think functionally it behaves the same, but we should just see if this is a restore side issue or a case we handle here.